### PR TITLE
Multimasterformal

### PR DESCRIPTION
**Pull request by kavyasreedhar**
This PR makes small changes to avoid errors with unused and unconnected signals that Nestan faced with SMT and Joey faced when running end to end tests with AHAFlow and Garnet. There should be no functional changes.

Update: all memory_core and interconnect tests in Garnet pass with this PR.

### CODE REVIEW

File: lake/top/lake_top.py [hunk 1]
```diff
@@ -357,7 +357,7 @@ def __init__(self,
         self.wire(cfg_seq_clk, kts.util.clock(self._gclk))
 
         self.add_child(f"config_seq", stg_cfg_seq,
-                       clk=self._cfg_seq_clk,
+                       clk=self._gclk,
```
Comment (mbstrange2) [inline lake/top/lake_top.py:360]: Why did you change this? These actually have a different clock - you can't write to the SRAM at configuration time with this different clock...

Comment (kavyasreedhar) [reply lake/top/lake_top.py:360]: I changed it because there were issues with having 2 clocks for one of the scripts Nestan was using, but we ended up not using that anyway, so I will revert it.

Comment (kavyasreedhar) [reply lake/top/lake_top.py]: Just pushed, but isn't cfg_seq_clk just connected to gclk anyway? Does this make a difference?

**Review by mbstrange2 — APPROVED** [review]
LGTM
